static/assets/js/core.js

```diff
@@ -12,6 +12,31 @@ let hexToRgba = function(hex, opacity) {
   return 'rgba(' + rgb.r + ', ' + rgb.g + ', ' + rgb.b + ', ' + opacity + ')';
 };
 
+let setCookie = function(name, value, days) {
```

Comment:
This `core.js` is actually from the template (https://tabler.io/) so I think it'd be better to put custom code in a different file and include it in `base.html`.